assembl/tests/fixtures/vote_session.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+
+@pytest.fixture(scope="function")
+def vote_session(request, test_session, discussion, timeline_vote_session,
+                    simple_file, moderator_user):
+    from assembl.models import VoteSession, VoteSessionAttachment, LangString
+    vote_session = VoteSession(
+        discussion_phase=timeline_vote_session,
+        title=LangString.create(u"vote session fixture", "en"),
+        sub_title=LangString.create(u"vote session sub title fixture", "en"),
+        instructions_section_title=LangString.create(u"vote session instructions title fixture", "en"),
+        instructions_section_content=LangString.create(u"vote session instructions fixture. Lorem ipsum dolor sit amet", "en"),
+        propositions_section_title=LangString.create(u"vote session propositions section tile fixture", "en")
```

Comment:
Why do you write fixture in each string? I find it weird. Here you wrote tile btw.

Reply:
So we can instantly know this is a fixture if this data appears somewhere